src/sass/components/_header.scss

```diff
@@ -222,3 +222,85 @@ header {
     }
   }
 }
+
+@media (max-width: 768px) {
+  header {
+    .top-bar {
+      height: 45px;
+
+      ul li {
+        line-height: 45px;
+      }
+
+      .text-left ul li {
```

Comment:
Nie odwołuj się do klas generycznych - lepiej dodaj jakąś, żebyś mógł ją identyfikować. Bo jeśli ktoś zmieni na text-right to wszystko Ci się sypnie